modules/sample-akkaHttp/src/test/scala/core/issues/Issue148.scala

```diff
@@ -153,7 +153,7 @@ class Issue148Suite extends FunSuite with Matchers with EitherValues with ScalaF
           .toEntity
       ) ~> route ~> check {
       rejection match {
-        case MalformedFormFieldRejection("foo", message, _) => message shouldBe "expected json value got b (line 1, column 1)"
+        case MalformedFormFieldRejection("foo", message, _) => message shouldBe "expected json value got 'blep' (line 1, column 1)"
```

Comment:
This is somewhat worrisome; Do you have an context on why this changed?

Reply:
I think it's the change in Circe version